modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/dialog/OptionsDialog.kt

```diff
@@ -37,6 +39,32 @@ class OptionsDialog : BottomSheetDialogFragment() {
     private var onDismiss: (() -> Unit)? = null
     private var forceDarkTheme: Boolean = false
 
+    companion object {
+        private const val KEY_FORCE_DARK_THEME = "force_dark_theme"
+        private const val KEY_TITLE = "title"
+        private const val KEY_OPTIONS = "options"
+        private const val KEY_ICON_COLOR = "iconColor"
+    }
+
+    @RequiresApi(Build.VERSION_CODES.TIRAMISU)
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        savedInstanceState?.let {
+            options.addAll(it.getParcelableArrayList(KEY_OPTIONS, OptionsDialogOption::class.java) ?: mutableListOf())
```

Comment:
This works fine on phones running Android 13, but any phones running older versions of Android will crash when they try to call the `getParcelableArrayList` method that only exists on Android 13. We need to do a version check and call the appropriate method for both Android 13 and for earlier versions. I would suggest creating an extension method similar to what we did with [getSerializableCompat](https://github.com/Automattic/pocket-casts-android/blob/aa9ae49b05e94fae69cdf7c1aa71241a8e0120f9/modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/extensions/Bundle.kt#L1-L0).